### ISSUE TITLE
vifm as an nvim cder

### ISSUE THREAD

**Issue by MikeLemo1** [vifm/vifm.vim#94]
I'm lookings to make something like this telescope plugin: [Zane-/cder.nvim](https://github.com/Zane-/cder.nvim) but with the vifm plugin inside nvim and I guess if there's no simple way to pass as "execute command mode callback function" something like `:EditVifm exc_callback="cwd %result_dir"` or the other option is as an nvim plugin but before diving into thos adventure I wanted to ask here wether there's a simple way to develop with vifm as an file explorer tool to performs nvim commands and actions?

Any tips and tricks?

**Comment by xaizek** [comment]
Sounds like you want the same as in #86?

**Comment by MikeLemo1** [comment]
Yup joining this question didn't notice this thread, I even ahd the feeling I was parasitic posting this suggestion/help in the issues thread
